sunpy/tests/figure_hashes_mpl_321_ft_261_astropy_401post1.json

```diff
@@ -24,19 +24,18 @@
     "sunpy.map.tests.test_plotting.test_plot_rotated_aia171": "4c96dbf9b4374283b9669c713a95fbd0cd654dfc75d293dd0c99073ce1a0039e",
```

Comment:
why the changes to the hashes? Including one going missing?!

Reply:
That's what the CI reports.